Create daleth.cafe.md (#278)
deltachat/provider-db: _providers/daleth.cafe.md

---
name: daleth.cafe
status: OK
domains: 
  - daleth.cafe
server:
  - type: IMAP
    socket: TLS
    hostname: daleth.cafe
    port: 993
    username_pattern: EMAIL
  - type: SMTP
    socket: STARTTLS
    hostname: daleth.cafe
    port: 587
    username_pattern: EMAIL
strict_tls: true
last_checked: 2024-02
skip_auto_test: false
website: https://nein.jetzt
---

> This ist a chatmail-instance with docker byside. 

Reviewer: ```suggestion
This ist a chatmail-instance with docker running in parallel. 
```

Then let's use this?

---

> name: Csl. Thomas Lüchow
> status: Teacher and Coach for Adults

Reviewer: ```suggestion
name: daleth.cafe
status: OK
```

This is the name of the instance, not of the person - I'd use daleth.cafe. Also because the name is a double with mehl.cloud otherwise.

The name is what's shown on the start page of https://providers.delta.chat/

That status needs to be `OK`, so we can show in the app that the server is spelled correctly.

domains: 
  - daleth.cafe
server:
  - type: IMAP
    socket: TLS
    hostname: daleth.cafe
    port: 993
    username_pattern: EMAIL
  - type: SMTP
    socket: STARTTLS
    hostname: daleth.cafe
    port: 587
    username_pattern: EMAIL

> before_login_hint: Wellcome to mehl.cloud
> after_login_hint: Now you are loged in

Reviewer: ```suggestion
```

I would leave out those two lines, they are only meant for giving users tips on how to configure the server. With chatmail, this is not needed.

strict_tls: true

> last_checked: 

Reviewer: ```suggestion
last_checked: 2024-02
```

let's add the current month.

skip_auto_test: false
website: https://nein.jetzt
---

> This ist a chatmail-instance with docker byside. 

Reviewer: Hm, meinst du es läuft daneben noch docker? Oder läuft chatmail in docker? `byside` ist so kein englisches Wort, vllt kann ich weiterhelfen?

Author: **lets use 'parallel'**

Some additional services are usefull, but the most important reason is to load bot as a docker container. 

I'am testing two ways to get some services parallel to chatmail. on Mehl.cloud I edited the nginx.conf, on daleth.cafe I have installed docker on a second IP-Address.
